app/src/main/java/edu/uc/zhukv/droneradarmap/dao/IRetrofitWeatherDAO.kt

```diff
@@ -1,6 +1,6 @@
-package dao
+package edu.uc.zhukv.droneradarmap.dao
 
-import dto.CurrentWeather
+import edu.uc.zhukv.droneradarmap.dto.CurrentWeather
```

Comment:
Multiple DTO and DAO Folders so I combined them inside the app directory. 

Rationale: I think it is good to keep like files in one file folder instead of having multiple DAO/DTO/Services in multiple folders.

Reply:
Ah, kind of.
1) Package names should not be specific to a developer, so removing that is good.
2) But they should also be unique, and are often a domain name in reverse.  That is missing.
3) They should not have uppercase letters.

So, while you're in here fixing #1, best to fix #2 and #3 as well.